website/docs/docs/explore/navigate-dbt-insights.md

```diff
@@ -24,31 +24,24 @@ The query console is the main component of <Constant name="query_page" />. It al
-  - ✨ Coming soon: Ability to open the [<Constant name="cloud_ide" />](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud) or [<Constant name="visual_editor" />](/docs/cloud/canvas) to continue editing your SQL query.
+- [Query console menu](#query-console-menu), which contains **Bookmark (icon)**, **Develop**, and **Run** buttons. 
 - [Query output panel](#query-output-panel), below the query editor and displays the results of a query:
   - Has three tabs: **Results**, **Details**, and **Chart**, which allow you to analyze query execution and visualize results.
 - [Query console sidebar menu](#query-console-sidebar-menu), which contains the **<Constant name="explorer" />**, **Bookmark**, **Query history**, and **<Constant name="copilot" />** icons.
 
 <Lightbox src="/img/docs/dbt-insights/insights-main.png" title="dbt Insights main interface with blank query editor" />
 
 ### Query console menu
-The Query console menu is located at the top right of the Query editor. It contains the **Bookmark** and **Run** buttons:
+The Query console menu is located at the top right of the Query editor. It contains the **Bookmark**, **Develop**, and **Run** buttons:
 
-<!--- add the below once ide and canvas support is fully released. 
-
-- Ellipsis **(`...`)** button &mdash; Access to develop in the [IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud) or [Canvas](/docs/cloud/canvas). Note, if you're on a tab with a bookmarked query, you can also click on the ellipsis button to **Share**, **Develop in the IDE**, **Edit details**, and **Delete** a bookmark.
-- -->
 - **Bookmark** button &mdash; Save your frequently used SQL queries as favorites for easier access.
   - When you click **Bookmark**, a **Bookmark Query Details** modal (pop up box) will appear where you can add a **Title** and **Description**.
   - Let [<Constant name="copilot" />](/docs/cloud/dbt-copilot) do the writing for you &mdash; use the AI assistant to automatically generate a helpful description for your bookmark.
   - Access the newly created bookmark from the **Bookmark** icon in the [Query console sidebar menu](#query-console-sidebar-menu). 
+ - **Develop**: Open the [<Constant name="cloud_ide" />](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud) or [<Constant name="visual_editor" />](/docs/cloud/canvas) to continue editing your SQL query.
 - **Run** button &mdash; Run your SQL query and view the results in the **Results** tab.
-- ✨ Coming soon &mdash; Ability to access the [<Constant name="cloud_ide" />](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud) or [<Constant name="visual_editor" />](/docs/cloud/canvas) from the Query console menu to develop your query and promote it to a reusable model.
 
-<!-- replace image when new UI changes are released. might use a gif here instead of jpg img  
-<Lightbox src="/img/docs/dbt-insights/insights-ellipsis.jpg" title="dbt Insights ellipsis button" />
--->
+  <Lightbox src="/img/docs/dbt-insights/develop-menu.jpg" title="dbt Insights Develop menu." />
```

Comment:
<img width="639" alt="image" src="https://github.com/user-attachments/assets/1c1a0b09-ccdf-48f1-917d-64fc96f31bc1" />

not loading in the preview!